routes/notesRoutes.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const helper = require('../data/helper.js');
+const router = express.Router();
+
+router
+    .route('/')
+    .get(async (req, res, next) => {
+        try {
+            const notes = await helper.getNotes();
+            res.status(200).json(notes);
+        } catch (err) {
+            next(err);
+        }
+    })
+    .post(async (req, res, next) => {
+        const note = req.body;
+        try {
+            const count = await helper.addNote(note);
+            res.status(201).json({
+                id: count[0]
+            });
+        } catch (err) {
+            next(err);
+        }
+    });
+
+router
+    .route('/:id')
+    .get(async (req, res, next) => {
+        const {
+            id
+        } = req.params;
+        try {
+            const result = await helper.getNote(id);
+      const note = result[0];
+      note ? res.status(200).json(note) : next({ statusCode: 404 });
+        } catch (err) {
+            next(err);
+        }
+    })
+    .put(async (req, res, next) => {
+        const {
+            id
+        } = req.params;
+        const note = req.body;
```

Comment:
Destructure the title and content from the request body. If you take all of the data, it may include malicious attacks on your database.